packages/block-library/src/common.scss

```diff
@@ -81,3 +81,34 @@
 .items-justified-space-between {
 	justify-content: space-between;
 }
+
+.screen-reader-text {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	-webkit-clip-path: inset(50%);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+}
+
+.screen-reader-text:focus {
+	background-color: $gray-300;
+	clip: auto !important;
+	clip-path: none;
+	color: #444;
+	display: block;
+	font-size: 1em;
+	height: auto;
+	left: 5px;
+	line-height: normal;
+	padding: 15px 23px 14px;
```

Comment:
The font size, height, left property, padding, these all seem very optimized for a specific context. Let me see if I can provide something better.

Reply:
Here's how it looks now:

<img width="755" alt="Screenshot 2021-03-24 at 08 48 26" src="https://user-images.githubusercontent.com/1204802/112273736-fa1df500-8c7d-11eb-97e4-5420101b88c3.png">

Here's how it looks with a few tweaks:

<img width="643" alt="Screenshot 2021-03-24 at 08 50 00" src="https://user-images.githubusercontent.com/1204802/112273752-fe4a1280-8c7d-11eb-929e-aedd1c5229fb.png">

That's with this CSS:

```
background-color: white;
clip: auto !important;
-webkit-clip-path: none;
clip-path: none;
color: black;
display: block;
font-size: 1em;
height: auto;
left: 1em;
line-height: normal;
padding: 1em;
text-decoration: none;
top: 1em;
width: auto;
z-index: 100000;
```

So most of the px values are substituted for 1em, and it uses white and black. What do you think Riad?

Reply:
Updated.